### PR TITLE
Tuning capabilities & `query_name` fix

### PR DESCRIPTION
**Pull request by jointsimon**
* Adds tuning capabilities via `NUM_WORKERS` and `BATCH_SIZE`
* Fixes `query_name` would only capture parameterized query names. It'll now capture non-parameterized queries.
* Fixes frequent "connection lost to ES"

### CODE REVIEW

File: config/logstash.yml [hunk 1]
```diff
@@ -2,11 +2,9 @@ http.host: "0.0.0.0"
...
 
 pipeline:
+  workers: ${NUM_WORKERS:1}
   batch:
     # The maximum number of events an individual worker thread
     # will collect from inputs before attempting to execute
     # its filters and outputs.
-    size: 10
-
-xpack.monitoring.enabled: false
```
Comment (jointsimon) [inline config/logstash.yml:11]: X-pack is not available on Aptible's ES

File: config/logstash.config [hunk 2]
```diff
@@ -50,21 +50,23 @@ filter {
...
     hosts => "${ELASTICSEARCH_URL}"
-    timeout => 1
-    sniffing => false
     pool_max => 1
-    template_name => "ja-logstash"
+    sniffing => false
     template => "/usr/share/logstash/index_template/ja_index_template.json"
+    template_name => "ja-logstash"
     template_overwrite => "true"
+    validate_after_inactivity => 200
```
Comment (jointsimon) [inline config/logstash.config:70]: Fixes frequent "connection lost to ES"

Comment (niclas-w) [reply config/logstash.config:70]: Well, that sound good. Doesn't it?

File: config/logstash.yml [hunk 3]
```diff
@@ -2,11 +2,9 @@ http.host: "0.0.0.0"
...
-xpack.monitoring.elasticsearch.url: $ELASTICSEARCH_URL
+    size: ${BATCH_SIZE:10}
```
Comment (niclas-w) [inline config/logstash.yml:10]: Is the batch size measured in number of messages? Maybe we should try to increase it to reduce overhead?

**Review by niclas-w — APPROVED** [review]
I have to trust Mr Elk here =)

LGTM!

File: config/logstash.yml (hunk 3 above)
Comment (jointsimon) [reply config/logstash.yml:10]: Yeah, I had 10 when I debugged during the ELK crisis.. gonna bump it up again!

**Pull request merged**